API/src/main/java/xyz/kyngs/librelogin/api/server/ServerHandler.java

```diff
@@ -29,6 +29,19 @@ public interface ServerHandler<P, S> {
      */
     ServerPing getLatestPing(S server);
 
+    /**
+     * Chooses an optimal lobby server to connect the player to. Usually the one with the lowest player count.
+     *
+     * @param user     The user of the player, or null if the player is from bedrock
+     * @param player   The player we're choosing the server for
+     * @param remember Whether to respect the remember last server option
+     * @param fallback Whether to select fallback server or not
+     * @throws java.util.NoSuchElementException If there is no lobby available
```

Comment:
Why throw NoSuchElementException if the original behavior was to return null? This is an unnecessary breaking change.

Reply:
Fixed this in [52d3deb](https://github.com/kyngs/LibreLogin/pull/103/commits/52d3deba0498437031f935272e925f3b95bccb65)